src/Endpoints/Delegates/HandlesMultiSearch.php

```diff
@@ -13,14 +13,16 @@ trait HandlesMultiSearch
     /**
      * @param list<\Meilisearch\Contracts\SearchQuery> $queries
      */
-    public function multiSearch(array $queries = [])
+    public function multiSearch(array $queries = [], array $multiSearchParams = [])
```

Comment:
would be nice to define `$multiSearchParams` shape

Reply:
Hey @norkunas what do you mean by `$multiSearchParams shape`?

Reply:
hey. i mean what is possible to provide in it, so users would get autocomplete from IDE

Reply:
also `\Meilisearch\Contracts\SearchQuery` should be imported with use statement instead of using FQCN in `@param`

Reply:
> i mean what is possible to provide in it

By introducing a new object like I did with the `FederationOptions` or a comment would be enough?

Reply:
I looked at the notion and as I see federation options has to be included alongside the queries, so not sure why FederationOptions was added to SearchQuery?

Reply:
The `federation` field is included alongside the queries, but then the `FederationOptions` is included in each search query and currently only contains a `weight` to make one query more important than another.
But in the future, they should contain more parameters.

Reply:
If they can differ then I suggest to add `MultiSearchFederation` with `limit`/`offset`/`reranking` and other available options that I don't know and document the `$multiSearchParams` as: `array{federation: MultiSearchFederation}`

Reply:
Seems good to me, will work on it later